### PR TITLE
Don't use the name SCONEPRO

### PR DESCRIPTION
**Pull request by mirjak**
I think it's better to just talk about a new QUIC version and to not make people believe this is the one and only proposal for a new protocol.

### CODE REVIEW

**Comment by mjoras** [comment]
@mirjak are you okay with the draft name having SCONEPRO? Fwiw I also think if any of this is taken up as a WG the entire name is up for re litigation

**Comment by mirjak** [comment]
I think you need to have sconepro in the draft name to make sure the datatracker correctly show it with the BoF. I agree this might anyway chance with a wg but for now we need to use the BoF name. We can easily rename after the BoF if people want to continue to work on this draft.

**Comment by mirjak** [comment]
As a side note, I extended the PR now to also cover the second part of the doc (sorry stopped reading to early).

**Comment by mjoras** [comment]
@mirjak can you fix the lint the check is mad about (line wrapping I think?) then I can merge.

File: draft-joras-sconepro-quic-protocol-alt.md [hunk 1]
```diff
@@ -211,24 +211,24 @@ Host:
...
+client opportunistically sends a QUIC long header packet destined to the same
+endpoint IP address and port using this new version. This packet can be be parsed by any
+capable network element on the path that support this new QUIC version. These elements
 MAY forward these packets in the normal fashion, such that all
-SCONEPRO-capable devices can see its contents. All SCONEPRO-capable elements
-are able to respond in a similar fashion, by creating their own SCONEPRO QUIC
-packets and sending it to the SCONEPRO QUIC client matching the IP/port tuple
-being utilized by the end to end QUIC connection.
+capable devices on path can see its contents. All capable elements
+are able to respond in a similar fashion, by creating their own long header
```
Comment (mirjak) [inline draft-joras-sconepro-quic-protocol-alt.md]:
```suggestion
are able to respond in a similar fashion, by creating their own long header
```

**Comment by mirjak** [comment]
Done but note this is a PR on there alternative md Marcus provided on Friday

**Pull request merged**